### PR TITLE
Feature/animatable barviews

### PR DESCRIPTION
**Pull request by gkye**

### CODE REVIEW

File: IBAnimatableMaterial/AnimatableMaterialView.swift [hunk 1]
```diff
@@ -0,0 +1,129 @@
+//
+//  AnimatableMaterialView.swift
+//  IBAnimatableMaterial
+//
+//  Created by George Kye on 2016-07-24.
+//  Copyright © 2016 IBAnimatable. All rights reserved.
+//
+
+import Foundation
```
Comment (JakeLin) [inline IBAnimatableMaterial/AnimatableMaterialView.swift:9]: Double up

**Comment by JakeLin** [comment]
Cool, thanks 👍

File: IBAnimatableMaterial/AnimatableMaterialView.swift (hunk 1 above)
Comment (gkye) [inline IBAnimatableMaterial/AnimatableMaterialView.swift:9]: Hmm, not too sure what this means :disappointed:

Comment (JakeLin) [inline IBAnimatableMaterial/AnimatableMaterialView.swift:9]: You have one at line 11 already  😁

Comment (gkye) [inline IBAnimatableMaterial/AnimatableMaterialView.swift:9]: Fixed :smiley_cat:

Comment (JakeLin) [inline IBAnimatableMaterial/AnimatableMaterialView.swift:9]: 👍

**Pull request merged**